File: app/src/main/java/com/s15752/CalculatorBMI/DishRecommendation.java
package com.s15752.CalculatorBMI;

public class DishRecommendation {
    private static float NORMAL_DOWN_VALUE = 17.2f;
    private static float NORMAL_TOP_VALUE = 30.0f;

    public DishRecommendation() {}


    public String dishRecommendation(float bmi)
    {
        if (bmi < NORMAL_DOWN_VALUE) {
        return "You can eat cheeseburger";
        } else if (NORMAL_DOWN_VALUE < bmi && bmi < NORMAL_TOP_VALUE) {
            return "You can eat cheeseburger, but with carrot and salat!";
        } else {
            return "You can eat only vegetables..";
        }
    }
}

File: app/src/main/java/com/s15752/CalculatorBMI/DishRecommendation.kt
package com.s15752.CalculatorBMI

class DishRecommendation {
    fun dishRecommendation(bmi: Float): String {
        return if (bmi < NORMAL_DOWN_VALUE) {
            "You can eat cheeseburger"
        } else if (NORMAL_DOWN_VALUE < bmi && bmi < NORMAL_TOP_VALUE) {
            "You can eat cheeseburger, but with carrot and salat!"
        } else {
            "You can eat only vegetables.."
        }
    }

    companion object {
        private const val NORMAL_DOWN_VALUE = 17.2f
        private const val NORMAL_TOP_VALUE = 30.0f
    }
}